cypress/integration/InstancesTest.ts

```diff
@@ -1,8 +1,39 @@
+import { countBy } from "cypress/types/lodash";
 import { format } from "date-fns";
+import { deleteInstance } from "./DeleteTest";
```

Comment:
it would be better to have some `DeleteUtil` file, or maybe some general `Delete` cusotm cypress command, that would be able to delete anything? instance, source, action ?

Reply:
@jomarko @sjamboro 
I created Util.ts file for common functions like CreateInstance, DeleteInstance.
For now, I accounted for Instances only but we can make this more generic like 'Create or Delete anything" in future